### DISCUSSION TITLE
Swap file usage

### DISCUSSION THREAD

**Discussion by OneScuzzie2k** [blakeblackshear/frigate#14003] [Ask A Question]
I have Frigate running on Proxmox, it is set to have 2 CPU, 2GB Ram, which is using about 1.25, I had the swap file set to 512mb it maxed out and kept crashing, I have now set to 1gb and its at 89.57% and still growing? any suggestions? I am using a Coral TPU PCI, and Intel GPU Drivers installed. I have 10 Cameras.

**Comment by hawkeye217** [comment]
There are known issues with the Proxmox kernel that cause memory leaks when using Frigate. See this discussion: https://github.com/blakeblackshear/frigate/discussions/10921

**Comment by OneScuzzie2k** [comment]
The memory usage is fine, its the swap file, and I am not using in a docker.

**Reply by hawkeye217** [reply]
This is something OS related, Frigate doesn't manage the swap file separately from memory.

**Reply by OneScuzzie2k** [reply]
> This is something OS related, Frigate doesn't manage the swap file separately from memory.

as I said memory usage is fine, not a issue, sat nicely at 54% the Swap file is what is growing every minute until it crashes

**Reply by hawkeye217** [reply]
Right, frigate doesn't know the difference between memory and swap. So this is not something that Frigate can fix.

**Reply by NickM-27** [reply]
You can set a swappiness along with other variables on the docker container in the docker run / compose

https://docs.docker.com/engine/containers/resource_constraints/#--memory-swappiness-details

**Reply by OneScuzzie2k** [reply]
> You can set a swappiness along with other variables on the docker container in the docker run / compose
> 
> https://docs.docker.com/engine/containers/resource_constraints/#--memory-swappiness-details

As I stated I am not using Docker..... I have set the swap file to 0 on the container now and will monitor
